src/_components/ContactUs.jsx

```diff
@@ -13,6 +13,11 @@ export default ({ comp, title }) => {
           style="px-16 py-2 rounded-lg bg-transparent border-[1px] border-[#00070E] text-[#00070E] font-semibold text-lg inline-block max-lg:px-16 max-lg:py-6 max-lg:text-3xl"
           redirect_url="mailto:hello@sitblueprint.com"
```

Comment:
can you change this to sit.blueprint@gmail.com, the hello@sitblueprint.com isn't really monitored